x-pack/plugin/security/lib/nimbus-jose-jwt-fixed/build.gradle

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
nit: can we call this `modified` instead of `fixed` ? Fixed implies we fixed a bug, but really we just modified the original to support the security manager.